pkg/ed25519/ed25519.go

```diff
@@ -193,7 +205,10 @@ func Verify(publicKey PublicKey, message, sig []byte) bool {
 	kh.Write(message)
 	hramDigest := make([]byte, 0, sha512.Size)
 	hramDigest = kh.Sum(hramDigest)
-	k := edwards25519.NewScalar().SetUniformBytes(hramDigest)
+	k, _ := edwards25519.NewScalar().SetUniformBytes(hramDigest)
```

Comment:
```suggestion
	k, err := edwards25519.NewScalar().SetUniformBytes(hramDigest)
```
I assume the `err` is missing here.